database/database.go

```diff
@@ -359,3 +316,52 @@ func (db DatastoreBacked) DeleteIneligibleDomainStates(domains []string) (err er
 	}
 	return delete(keys)
 }
+
+// GetAllIneligibleDomainStates returns all the ineligible domains in the database
+func (db DatastoreBacked) GetAllIneligibleDomainStates() (states []IneligibleDomainState, err error) {
```

Comment:
Do you think this function can be split into a separate PR? I don't think it interacts with the rest of this PR

Reply:
I'm not sure what's wrong with Github's tracking. This function was already in the database.go file -- the only functions I added were StatesForDomains and SetPendingAutomatedRemoval(). Not really sure how to fix this, so would it be a problem if it was included in this pr?